### DISCUSSION TITLE
Cannot find sync'ed files on Android

### DISCUSSION THREAD

**Discussion by jphbutler64** [opengisch/QField#3022] [Q&A]
The files that are sync'ed from QField Cloud on Android do not appear in the standard Android/data/ch.opengis.qfield tree. Where can they be?
I want to access the storage location so that I may add manually some very large static files that do not travel well over the network (working from a remote field location in Africa), and which probably exceed the allowance on the QField Cloud server anyway. I patched a QGIS Cloud download using that method on Windows (QGIS 3.24.0), and it worked perfectly; now I want to do the same on Android, but cannot find the directory.

**Comment by nirvn** [comment]
@jphbutler64 , in your case, you are highly advised to rely on localized data paths, which allows you to place large datasets (vector and/or raster) into locations that are independent from the project file and will be access to open those localized data paths across multiple devices / desktop environments.

Not the greatest name for this feature, I know :). You can read some documentation on this here: https://docs.qfield.org/how-to/outside-layers/
